src/components/layout/Banner.jsx

```diff
@@ -0,0 +1,64 @@
+import './Banner.css';
+import { useState, useEffect } from 'react';
+
+export default function Banner() {
+  const [showBanner, setShowBanner] = useState(true);
+  const [bannerData, setBannerData] = useState([]);
+
+  function handleExitBanner() {
+    setShowBanner(false);
+  }
+
+  useEffect(() => {
+    fetch('/amanuensis/notifications', {
```

Comment:
Currently, the rest of the app is structured as a [Redux toolkit](https://redux-toolkit.js.org/tutorials/quick-start) app and follows the conventions of organizing all API calls and data management into Redux slices. For cohesion with the rest of the app would probably be best if you created a Redux slice for the message center and include this API call there.